src/fsharp/NicePrint.fs

```diff
@@ -1121,14 +1121,20 @@ module private PrintTypes =
         ty.GetAssemblyName()
 
 /// Printing TAST objects
-module private PrintTastMemberOrVals = 
-    open PrintTypes
-    let private prettyLayoutOfMember denv typarInst (v: Val) = 
+module private PrintTastMemberOrVals =
+    open PrintTypes 
+    let private prettyLayoutOfMemberShortOption denv typarInst (v:Val) short =
```

Comment:
Since this is a private function I think it's fine to leave the name as-is and just have the additional flag. Perhaps rename the flag to `shortMemberDisplay`?

Reply:
Ok will do. Thanks :)